package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "mmm-googlecalendar",
-  "version": "1.0.0",
```

Comment:
Bumping by 1 seems excessive for this fix, what about `1.0.1` ?

Reply:
I agree. I'm not sure why that version number automatically bumped up.